apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/dokument/DokumentRoute.kt

```diff
@@ -25,6 +29,13 @@ fun Route.dokumentRoute(safService: SafService, behandlingKlient: BehandlingKlie
             }
         }
 
+        post("{journalpostId}/ferdigstill") {
+            val request = call.receive<FerdigstillJournalpostRequest>()
+            dokarkivService.ferdigstill(call.parameters["journalpostId"]!!, request)
+
+            call.respond(HttpStatusCode.OK)
```

Comment:
Burde denne returnere Created?

Reply:
Tja, vet ikke helt om det blir riktig. Created er så vidt jeg vet myntet på respons når noe er _opprettet_. Ferdigstilling av journalpost bare _oppdaterer_ en allerede eksisterende journalpost.  🤔

Reply:
Sant nok, så da er det kanskje http-verbet eg synes blir feil, for post er jo mest for å opprette, ikkje oppdatere. Ikkje at vi har vore konsekvente på det, men det kan vi jo godt starte å bli.

Reply:
Enig. Vi har aldri satt en standard for bruk på teamet, men burde sikkert gjøre det og rydde opp en dag.